superset/migrations/versions/c2acd2cf3df2_alter_type_of_dbs_encrypted_extra.py

```diff
@@ -39,7 +39,7 @@ def upgrade():
             batch_op.alter_column(
                 "encrypted_extra",
                 existing_type=sa.Text(),
-                type_=EncryptedType(sa.Text()),
+                type_=sa.LargeBinary(),
```

Comment:
Note that the underlying impl for `EncryptedType` is `LargeBinary`, so `EncryptedType` with no key is equivalent.

Reply:
Seems that we need to eventually migrate it: https://github.com/kvesteri/sqlalchemy-utils/blob/2e8ee0093f4a33a5c7479bc9aaf16d7863a74a16/sqlalchemy_utils/types/encrypted/encrypted_type.py#L455